pkgs/by-name/co/cosmic-tasks/package.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, git
+, pkg-config
+, wrapGAppsHook
+, atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk3
+, libsecret
+, libxkbcommon
+, openssl
+, pango
+, sqlite
+, vulkan-loader
+, stdenv
+, darwin
+, wayland
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-tasks";
+  version = "0-unstable-2024-04-19";
+
+  src = fetchFromGitHub {
+    owner = "edfloreshz";
+    repo = "cosmic-tasks";
+    rev = "e6d8539991e5e4ce3d01008442bac0aaa7445201";
+    hash = "sha256-iio0vjZBbUKJm6pC1iETZwQLloZYjnsiNqVWT9woJ/c=";
+    leaveDotGit = true;
```

Comment:
Why is `leaveDotGit` here?

Reply:
It checks something on the git tree during the compilation.
I added a comment.

Reply:
That sounds sus, shouldn't have to. Looks like you can set `VERGEN_GIT_COMMIT_DATE` and `VERGEN_GIT_SHA` to mitigate this. Should be obvious as to what these two envs should be set to.

Reply:
Indeed ! Done.